src/Client.cpp

```diff
@@ -996,3 +1006,32 @@ void CClient::ParseIdentifier(const CString& sAuthLine) {
 		m_sUser = sAuthLine;
 	}
 }
+
+void CClient::SetNetworkCaps(const SCString& ssCaps)
```

Comment:
There is some value in disabling caps on client side when server disconnects, and reenabling them again later when server connects. For client to clear cache of account-notify, so that upon reconnect of server, client can know that it knows nothing about who is on which account now.

Probably it doesn't matter for away-notify, because while network is disconnected, there are no channels.